docs/app/pages/Components/Chips/Chips.vue

```diff
@@ -53,6 +54,13 @@
       <code-example title="Duplicated Feedback" :component="examples['duplicated-feedback']" />
     </div>
 
+    <div class="page-container-section">
+      <h2>Formatter</h2>
+
+      <p>There could be a rule to be followed by the inserted chips:</p>
```

Comment:
A better message could be:

Reply:
fix 8a521664a13fc437f9876bfe458f1912f8fb1101